packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -23,7 +23,7 @@ type ColumnType = (typeof columnTypes)[number];
 const statusIcons = ['Group', 'InProgress'] as const;
 type StatusIcon = (typeof statusIcons)[number];
 
-// Options from https://react.carbondesignsystem.com/?path=/docs/components-tag--overview
+// Options from https://react.carbondesignsystem.com/?path=/docs/components-tag--overview plus orange for priority tags
 const carbonTagColors = [
```

Comment:
Maybe rename this to `priorityTagColors` since it's no longer a list of carbon tag colors?